.github/workflows/npm-audit-fix.yml

```
@@ -0,0 +1,27 @@
+name: npm audit fix
+
+on:
+  schedule:
+    - cron: 0 0 * * * # Runs at 00:00 UTC every day
+
+# on:
+#   push:
+#     branches:
+#       - 'master'
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v2.1.4
+      with:
+        node-version: '12.x'
+        registry-url: 'https://registry.npmjs.org'
+    - run: npm install -g "npm@^7"
+    - uses: ybiquitous/npm-audit-fix-action@v2.1.5
```

The v3 of this action supports npm 7 whereas the v2 version does not
```suggestion
    - uses: ybiquitous/npm-audit-fix-action@v3
```